_posts/2022-02-11-helm-multiple-worker-groups.md

```diff
@@ -0,0 +1,124 @@
+- [Optional] To have [kubectl](https://kubernetes.io/docs/tasks/tools/) installed.
+That’s it, let’s get started!
+
+## Install Dask Kubernetes
+From the [documentation](https://kubernetes.dask.org/en/latest/installing.html),
+```console
+pip install dask-kubernetes --upgrade
+```
+or
+```console
+conda install dask-kubernetes -c conda-forge
+```
+
+## Install the Dask Helm Chart
+First, deploy Dask on Kubernetes with Helm:
+```console
+helm repo add dask https://helm.dask.org/
+helm repo update
+helm install my-dask dask/dask
+```
+
+Now you should have Dask running on your Kubernetes cluster. If you have kubectl installed, you can run `kubectl get all -n default`
+<img src="/images/default-dask-cluster.png" alt="Default Dask Cluster Installed with Helm" width="661" height="373">
+
+You can see that we’ve created a few resources! The main thing to know is that we start with three dask workers. Remember, it doesn’t matter if you don't have kubectl installed because we’ll do something similar with HelmCluster 
+
+## Add GPU worker group to our Dask Deployment
+To create some GPU workers, we’ll need to change the default values in the Dask Helm Chart. To do this, we can create a copy of the current [`values.yaml`](https://github.com/dask/helm-chart/blob/main/dask/values.yaml), update it to add a GPU worker group and then update our helm deployment.  
+- First, you can copy the contents of the values.yaml file in the Dask Helm Chart and create a new file called my-values.yaml
+-  Next, we’re going to update the section in the file called ‘additional_worker_groups’. The section looks like this:
+```yaml
+additional_worker_groups: # Additional groups of workers to create
+# - name: high-mem-workers  # Dask worker group name.
+#   resources:
+#     limits:
+#       memory: 32G
+#     requests:
+#       memory: 32G
+# ... 
+# (Defaults will be taken from the primary worker configuration)
+```
+- Now we’re going to edit the section to look like this:
+```yaml
+additional_worker_groups: # Additional groups of workers to create
+- name: gpu-workers  # Dask worker group name.
+  replicas: 1
+  image:
+    repository: rapidsai/rapidsai-core
+    tag: 21.12-cuda11.5-runtime-ubuntu20.04-py3.8
+    dask_worker: dask-cuda-worker
+  extraArgs:
+    - --resources
+    - "GPU=1"
+  resources:
+    limits:
+      nvidia.com/gpu: 1
+```
+- Now we can update our deployment with our new values.yaml
+```console
+helm upgrade -f my-values.yaml my-dask dask/dask
+```
+- Again, if you have kubectl installed, you can run ‘kubectl get all -n default’, and you’ll see our new GPU worker pod running: `kubectl get all -n default`
+<img src="/images/gpu-worker-dask-cluster.png" alt="Dask Cluster Installed with Helm with a GPU worker" width="653" height="428">
+- Now we can open up a jupyter notebook or any editor to write some code.
+
+## Scaling the workers Up/Down
+```python
+from dask_kubernetes import HelmCluster
+cluster = HelmCluster(release_name="my-dask")
+cluster
+```
```

Comment:
It would be nice to have a little intro text here. Explain that we are going to import the `HelmCluster` cluster manager and point it to our Dask cluster. Perhaps go into a little detail about how `HelmCluster` can give quick access to logs, automatically port forwards the scheduler for you, etc.